packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js

```diff
@@ -165,10 +165,30 @@ const handleBackspaceKeyOnList = (editor, event) => {
   const [currentListItem, currentListItemPath] = Editor.parent(editor, editor.selection.anchor);
   const [currentList, currentListPath] = Editor.parent(editor, currentListItemPath);
   const isListEmpty = currentList.children.length === 1 && currentListItem.children[0].text === '';
+  const isNodeStart = Editor.isStart(editor, editor.selection.anchor, currentListItemPath);
+  const isEditorStart = Editor.before(editor, currentListPath) == null;
+  const isFocusAtTheBeginningOfALine = editor.selection.focus.offset === 0;
```

Comment:
You need to also check if it's the first child, in addition to the offset. Otherwise having modifiers only on part of the line can mess this up, like here:

https://github.com/strapi/strapi/assets/8087692/764ceeae-cf06-4020-981b-4f11a823b08f

Haven't tested but something like this could work:

```suggestion
  const isFocusAtTheBeginningOfALine = editor.selection.focus.offset === 0 && editor.selection.focus.at(-1) === 0;
```

Reply:
done